acceptance/config/beaker/options.rb

```diff
@@ -8,6 +8,7 @@
   "puppetservice" => 'puppetserver',
   "use-service" => true,
   "master-start-curl-retries" => 60,
-  "puppetpath" => "/etc/puppetlabs/agent"
+  "puppetpath" => "/etc/puppetlabs/puppet",
+  "puppetserver-confdir" => '/etc/puppetlabs/puppetserver/conf.d'
```

Comment:
The reason beaker should not have this information is because it's become clear that it's costly in terms of time and coordination overhead.  Better and more efficient to have each project provide the information to beaker, or have beaker dynamically inspect the information.  This will allow us to make these sorts of changes in the future without having to coordinate the changes to beaker across puppet, facter, hiera, puppetserver, etc...